components/scream/src/physics/cld_fraction/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+set(CIMEROOT ${SCREAM_BASE_DIR}/../../cime)
+list(APPEND CMAKE_MODULE_PATH ${CIMEROOT}/src/CMake)
+
+set(GENF90 ${CIMEROOT}/src/externals/genf90/genf90.pl)
+set(ENABLE_GENF90 True)
+include(genf90_utils)
+include(Sourcelist_utils)
+
+set(CLDFRAC_SRCS
+  atmosphere_cldfraction.cpp
```

Comment:
Picky, but: since you spell `cld_fraction` in the folder name (2 "words", as opposed to single "word" `cldfraction`), you might do the same in the file: `atmosphere_cld_fraction.cpp`.

Reply:
I've made these changes everywhere.  In the code should we also split the term?  In particular, should the namespace be called `cld_fraction` instead of `cldfrac`?

Reply:
I think so. It's more consistent.